lib/Arch/Runtime/HyperCall.cpp

```diff
@@ -386,6 +386,10 @@ Memory *__remill_sync_hyper_call(State &state, Memory *mem,
       mem = __remill_ppc_emulate_instruction(mem);
       break;
 
+  case SyncHyperCall::kPPCSysCall:
+      mem = __remill_ppc_syscall(mem);
```

Comment:
Unfortunately, I don't think we can implement this with inline asm like we've been doing for the x86 and ARM syscalls. The GCC/Clang inline assembly machine constraints for PowerPC don't seem to allow you to specify a particular register.